client/src/components/NewIssue/NewIssue.module.css

```diff
@@ -0,0 +1,5 @@
+.NewIssue {
```

Comment:
i think classnames use dash-case

Reply:
This is a **CSS Module** (not a CSS file). It is recommended to use camel casing in CSS Modules. 
Source: https://github.com/css-modules/css-modules#naming